src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -72,12 +78,15 @@ protected override void Dispose(bool disposing)
             }
             if (request.RequestUri.Scheme == UriSchemeHttps)
             {
+                // TODO: Add HTTPS support
                 throw NotImplemented.ByDesignWithMessage("HTTPS stack is not yet implemented");
             }
-            if (request.Content != null)
+            else if (request.RequestUri.Scheme != UriSchemeHttp)
```

Comment:
This is only checking UriSchemeHttp... does it also need to check UriSchemeHttps?